python-sdk/tutorials/automl-with-azureml/automl-nlp-multiclass/automl-nlp-text-classification-multiclass.ipynb

```diff
@@ -347,10 +355,6 @@
    "outputs": [],
    "source": [
     "automl_settings = {\n",
-    "    \"iterations\": 1,\n",
-    "    \"primary_metric\": \"accuracy\",\n",
-    "    \"max_concurrent_iterations\": 1,\n",
-    "    \"featurization\": \"auto\",\n",
```

Comment:
max_concurrent_iterations: this may be needed to invoke distributed?

Reply:
Yes that's correct. So I kept this parameter in multilabel example (which uses distributed training) only